packages/mobile/src/account/saga.ts

```diff
@@ -74,6 +87,30 @@ function* initializeAccount() {
   }
 }
 
+export function* fetchFinclusiveKyc() {
+  const personaKycStatus = yield select(kycStatusSelector)
+  const finclusiveKycStatus = yield select(finclusiveKycStatusSelector)
+
+  // If a user has already been approved through Persona KYC, then we need to see if Finclusive has accepted the user's KYC yet.
+  if (
+    personaKycStatus === KycStatus.Approved &&
+    finclusiveKycStatus !== FinclusiveKycStatus.Accepted
+  ) {
```

Comment:
I worry a bit about having this logic live in this layer. It would be surprising to me, as a consumer of this function, that the function wasn't actually fetching finclusive kyc if persona kyc was in a certain state

Reply:
moved.